style.css

```diff
@@ -19,6 +19,17 @@ header {
   width: 375px;
 }
 
+.error-message {
+  color: #fff;
+  font-weight: bold;
+  padding: 12px 8px;
+  font-size: 14px;
+  border-radius: 8px;
+  background-color: #ff0905ca;
```

Comment:
- [ ] Great job implementing this. Your method is top-notch for the error message. I want you to even do more by making sure that the background color of your error message blends with the design of your website. I think the current color is off. Using a stray color normally causes color conflict which is not a good UI approach I think. This color would have been better if you had a touch of red in your current design of the website. What do you think?
![image](https://user-images.githubusercontent.com/45668340/147144937-6512ce5d-677f-4acd-aa4e-366afb5e5aaa.png)